vslib/src/sai_vs_fdb.cpp

```diff
@@ -156,10 +206,21 @@ sai_status_t internal_vs_flush_fdb_entries(
              * data base.
              */
 
+            fi.bridge_port_id = it->second.at("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID")->getAttr()->value.oid;
+
+            SWSS_LOG_INFO("FDBDelNotification: fdb_entry:%s bpid:%s", sai_serialize_fdb_entry(fi.fdb_entry).c_str(), sai_serialize_object_id(fi.bridge_port_id).c_str());
+
             it = fdbs.erase(it);
+
+            sendFdbDelNotification(fi);
         }
     }
 
+    /* Sending notification for each fdb entry individually above, as a single
```

Comment:
please revert those changes, this remove in a loop can be enhanced adding lua script if necessary

Reply:
Sure, I will revert this.

Reply:
Hi Kamil, with this change reverted, per port and per vlan fdb flush will not work on VS.

Reply:
is that because your change ?

Reply:
> is that because your change ?

yes, per port and per vlan fdb flush support has been added in orchagent. As syncd does not support SAI_FDB_EVENT_FLUSHED for per port and per vlan, individual mac delete response is sent from VS instead of SAI_FDB_EVENT_FLUSHED.